src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,153 @@
 package core.basesyntax;
+//import jdk.internal.util.ArraysSupport;
+
+import java.util.Arrays;
+import java.util.NoSuchElementException;
 
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final Object[] EMPTY_ARRAYDATA = {};
```

Comment:
remove this variable, you don't need it

Reply:
and DEFAULT_CAPACITY? 
Array should have the default capacity,